src/components/Show/styles.css

```diff
@@ -0,0 +1,4 @@
+.meta {
+  color: gray;
+  padding-bottom: 20px;
```

Comment:
When categories are displayed below ShowHeader, this should be changed to padding-top

Reply:
I still think we should keep padding-bottom, as the category text becomes very close to the show content now.

![selection_rr](https://user-images.githubusercontent.com/21339571/46909513-160d0080-cf34-11e8-82ee-cbef0e9d5602.png)

Reply:
I added a 10px padding to top and 20px padding to bottom, looks better this way.